### ISSUE TITLE
[Question] Currently only for Windows environment.

### ISSUE THREAD

**Issue by JeepNL** [frankodoom/libwebp.net#2]
This is a great library. Super simple to use but with lots of options. I'm currently using FFmpeg to convert images to .webp on my Ubuntu VPS. Are you planning to support Linux env. as well?

**Comment by frankodoom** [comment]
Yes @JeepNL planning to support Linux very soon, you can find that in the project boards.

**Comment by JeepNL** [comment]
Wow! That's great. Looking forward to use your lib on linux!. And I've found the [projectboards ](https://github.com/frankodoom/libwebp.net/projects/1#card-66898250) now too 😉,

**Comment by cd-linxi** [comment]
Is Linux supported now?

**Comment by Nefcanto** [comment]
I'm also waiting for Linux support.

**Comment by vitvitskyi** [comment]
@frankodoom any updates on supporting Linux?
